packages/synapse-react-client/src/components/CardContainer/CardContainer.integration.test.tsx

```diff
@@ -119,7 +96,7 @@ describe('CardContainer tests', () => {
   beforeAll(() => server.listen())
   beforeEach(() => {
     jest.clearAllMocks()
-    server.use(...getHandlersForTableQuery(dataWithMultiplePagesFirstPage))
+    registerTableQueryResult(lastQueryRequest.query, syn16787123Json)
```

Comment:
Updated all tests that used `getHandlersForTableQuery`. Now, these handlers are already registered when you call `server.use`

To mock a table query, you must now call `registerTableQueryResult` which binds a particular `Query` to a `QueryResultBundle`. With this change, we can now use msw to mock multiple queries for a particular table.